src/main/scala/org/globalforestwatch/layers/Agc2000.scala

```diff
@@ -2,6 +2,11 @@ package org.globalforestwatch.layers
 
 import org.globalforestwatch.grids.GridTile
 
-case class Agc2000(gridTile: GridTile, model: String = "standard") extends FloatLayer with OptionalFLayer {
-    val uri: String = s"$basePath/gfw_aboveground_carbon_stock_2000/v20191106/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/Mg/geotiff/${gridTile.tileId}.tif"
```

Comment:
Need to change layer path to `s"$basePath/gfw_aboveground_carbon_stock_2000$model_suffix/v20191106/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/Mg_ha-1/geotiff/${gridTile.tileId}.tif"`